FirstProject/css/main.css

```diff
@@ -0,0 +1,90 @@
+header {
+    background-image: url(../img/feedback-banner.jpg);
+}
+
+.left_image {
+    height: 290px;
+}
+
+.right_description {
+    font-weight: bold;
+}
+
+
+.container-fluid {
+    background-color: lightgrey;
+}
+
+body {
+    overflow-x:hidden;
+}
+
+a {
+    text-decoration: none!important;
+    cursor: pointer;
+    color: black!important;
+    font-weight: 700;
+    font-size: 24px;
+}
+
+a:hover {
+    border-bottom: 4px solid blue;
+    border-bottom-width: 7px;
+}
+
+
+th, td {
+    padding-left: 55px!important;
+    font-size: 20px;
+}
+
+td {
+    border-bottom: 1px solid black;
+}
+
+
+footer.footer_section {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    text-align: center;
+}
+
+ul {
+    list-style-type: none;
+}
+
+li.bullets {
+    padding: 10px;
+}
+
+.submit_button {
+    float: right;
+    margin-top: 30px;
+    padding: 15px 40px;
+    margin-right: 100px;
+    border: 2px solid cornflowerblue;
+    color: white;
+    font-size: 16px;
+    background-color: cornflowerblue;
+    font-weight: bold;
+    border-radius: 4px;
+}
+
+form.language_form {
+    margin-top: 70px;
+}
+
+h2.description_section {
+    margin-left: 300px;
+}
+
+/*.row_section {*/
+    /*margin-left:254px;*/
+    /*margin-right: 255px;*/
+/*}*/
+
+fieldset.field_section {
```

Comment:
Because maybe in the future I will an some extra styling to fieldset_section class(or not) and I don't want to override some things. I want all to be very specific.